.github/renovate.json5

```diff
@@ -295,6 +295,53 @@
         "Dockerfile.clang-format"
       ],
       "enabled": false
+    },
+    {
+      // do not update dependencies from the replace directive from cilium/cilium
+      // see go.mod second replace directive comments for more info
+      "enabled": false,
+      "matchPackageNames": [
+        // All of these packages are maintained on a Cilium fork. Thus, we don't
+        // want to update them automatically.
+        "go.universe.tf/metallb",
+        "github.com/cilium/metallb",
+        "github.com/vishvananda/netlink",
+        "github.com/cilium/netlink",
```

Comment:
These two can be removed, as they are not replaced anymore (https://github.com/cilium/cilium/pull/27342 removes them from Cilium too).